ansible_collections/arista/avd/plugins/filter/add_md_toc.py

```diff
@@ -18,6 +18,59 @@
 except ImportError:
     HAS_MD_TOC = False
 
+DOCUMENTATION = r"""
+---
+name: add_md_toc
+collection: arista.avd
+author: Arista Ansible Team (@aristanetworks)
+version_added: "3.2"
+requirements:
+  - md_toc
+short_description: Parse the input MarkDown and add a Table of Contents between the toc_markers.
```

Comment:
short_description doesn't seem to match the preview site.

Reply:
not understood

Reply:
Turned out that another module had `add_md_toc` as the name as well, leading to duplicate docs and confusing this. Fixed it now.